nomos-cli/src/cmds/validator.rs

```diff
@@ -115,3 +124,28 @@ where
         .await
         .unwrap())
 }
+
+impl Reconstruct {
+    pub fn run(self) -> Result<(), Box<dyn std::error::Error>> {
+        tracing::subscriber::set_global_default(tracing_subscriber::FmtSubscriber::new())
```

Comment:
It would be awesome if tracing subscriber could be set before running the command, that way we would have one place to configure it, maybe reuse the nomos-tracing crate. Could this be added as a separate PR?

Reply:
I checked the tracing problem, and have no idea how to run something like that before anything runs ? cmd.run() is where it all starts, isn't it ? Perhaps @bacv, you might have better idea and faster hand to implement sandwich PR like we did for node config ?

Reply:
okay, will make a pr targeting this one.